csharp/src/Drivers/Apache/Spark/SparkStandardConnection.cs

```diff
@@ -219,6 +219,6 @@ internal override IArrowArrayStream NewReader<T>(T statement, Schema schema, IRe
 
         public override string AssemblyVersion => s_assemblyVersion;
 
-        protected override int ColumnMapIndexOffset => 0;
+        protected override int ColumnMapIndexOffset => 1;
```

Comment:
Can this be moved to `SparkConnection` and removed from both here and `SparkHttpConnection`?